### PR TITLE
Trigger fallback when locale value is empty

### PR DESCRIPTION
**Pull request by fyhring**
The translation fallback is only triggered when the translation doesn't exists. With this the locale translation must exists and not be empty.

### CODE REVIEW

File: resources/assets/sass/app.scss [hunk 1]
```diff
@@ -5147,11 +5147,6 @@ float: left;
   margin-left:10px;
 }
 
-.modal-open .select2-container{
```
Comment (fletch3555) [inline resources/assets/sass/app.scss:5150]: Why is this removed? Is it related to your change somehow?  If not, please revert this change

Comment (fletch3555) [reply resources/assets/sass/app.scss:5150]: @fyhring ^

Comment (fyhring) [reply resources/assets/sass/app.scss:5150]: Sorry. I made a mistake. I meant to make two diffent PR's. I'll try to revert.

Comment (fletch3555) [reply resources/assets/sass/app.scss:5150]: No worries.  Always do work in branches.  PRs aren't a "snapshot" of the changes at that point, they're a complete list of commits that differ between the source and destination branches and are therefore updated automatically if new commits are pushed.
